dask_sql/context.py

```diff
@@ -247,6 +248,15 @@ def create_table(
         if type(input_table) == str:
             dc.filepath = input_table
             self.schema[schema_name].filepaths[table_name.lower()] = input_table
+        elif "dask" in str(type(input_table)):
```

Comment:
If the function achieves your desired result, could you use `dd.utils.is_dataframe_like` since we use it in other places. That function might possibly include a wider positive result for dataframe like things you don't wish to include here however.

Reply:
Thanks!